docs/requirements-specification.md

```diff
@@ -57,4 +57,49 @@ _This content item details the use case overview of the system vision in its use
 
 ### UC5: Find expert in a field
 
-### UC6: Organizing capabilities
\ No newline at end of file
+### UC6: Organizing capabilities
+Only Create (C) use cases are explicitly defined here. Update (U) and Delete (D) need still to be defined and shall not contradict/invalidate Create use cases.
+
+| UC6a | Create capability group |
+|---|---|
+| Actors | Admin |If the capability group name exists already, the system prevents the Admin from creating the capability group. |
```

Comment:
The third cell ("If the capability group name exists already, the system prevents the Admin from creating the capability group.") is not rendered since the table only contains two columns.

Reply:
Ups. Copy/Pasta error.